File: 1 - INICIANTE/1045.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main() {
double a,b,c,x,y,z;
cin >> a >> b >> c;


if (b>=a && c>b) {
  y = a;
  a = c;
  c = y;
}

if (b>=a && c<=b) {
  x = a;
  a = b;
  b = x;
}

if (a>=b && a>c) {
  a = a;
  b = b;
  c = c;
}

if (b<c) {
  z = b;
  b = c;
  c = z;
}

//cout << a << "\n" <<  b << "\n" << c << "\n";


if (a>=(b+c)) {
  cout << "NAO FORMA TRIANGULO" << "\n";
  return 0;
}
if (pow(a,2)==(pow(b,2)+pow(c,2))) {
  cout << "TRIANGULO RETANGULO" << "\n";
}

if (pow(a,2)>(pow(b,2)+pow(c,2))) {
  cout << "TRIANGULO OBTUSANGULO" << "\n";
}

if (pow(a,2) < (pow(b,2)+pow(c,2))) {
  cout << "TRIANGULO ACUTANGULO" << "\n";
  }

if ((a==b)&&(b==c)&&(c==a)) {
  cout << "TRIANGULO EQUILATERO" << "\n";
}

if (((a==b)&&(c!=a)) || ((b==c)&&(a!=b))) {
  cout << "TRIANGULO ISOSCELES" << "\n";
}





return 0;
}

File: 1 - INICIANTE/python/1045.py
value = input().split(' ')
a, b, c = float(value[0]), float(value[1]), float(value[2])
lista = [a, b, c]
lista.sort(reverse=True) #ordenar decrescente
a, b, c = lista[0], lista[1], lista[2] #voltando as variaveis apenas pra ficar mais facil durante a escrita do if

if (a>=(b+c)):
    print('NAO FORMA TRIANGULO')
    exit(0)
if (a**2==(b**2+c**2)):
    print('TRIANGULO RETANGULO')
if (a**2>(b**2+c**2)):
    print('TRIANGULO OBTUSANGULO')
if (a**2<(b**2+c**2)):
    print('TRIANGULO ACUTANGULO')
if (a==b==c):
    print('TRIANGULO EQUILATERO')
if (a==b!=c) or (b==c!=a) or (c==a!=b):
    print('TRIANGULO ISOSCELES')
